num_of_consumers: 2
publish_voltage: True
technology: 2
design_capacity: 4.0
number_of_cells: 6
nominal_voltage: 24
full_charge_voltage: 25.2
cut_off_voltage: 18.0
internal_resistance: 0.11
current_filter_tau: 1
polarization_constant: 0.07
exponential_voltage: 0.7
exponential_capacity: 3.0
characteristic_time: 420
reversible_voltage_temp: 0.05
arrhenius_rate_polarization: 850
capacity_temp_coeff: 0.01
design_temperature: 25
temperature_response_tau: 0.5
